docs/content/en/docs/design/api.md

```diff
@@ -14,11 +14,11 @@ pipeline** and for **controlling the phases in the pipeline**.
 
 To retrieve information about the Skaffold pipeline, the Skaffold API provides two main functionalities:
   
-  * A [streaming event log]({{< relref "/docs/design/api#events-api">}}) created from the different phases in a pipeline run and
+  * A [streaming event log]({{< relref "#events-api">}}) created from the different phases in a pipeline run and
   
-  * A snapshot of the [overall state]({{< relref "/docs/design/api#state-api" >}}) of the pipeline at any given time during the run.
+  * A snapshot of the [overall state]({{< relref "#state-api" >}}) of the pipeline at any given time during the run.
 
-To control the individual phases of the Skaffold, the Skaffold API provides [fine grained control over]({{< relref "/docs/design/api#controlling-build-sync-deploy" >}})
+To control the individual phases of the Skaffold, the Skaffold API provides [fine grained control over]({{< relref "#controlling-build-sync-deploy" >}})
```

Comment:
```suggestion
To control the individual phases of the Skaffold, the Skaffold API provides [fine grained control]({{< relref "#controlling-build-sync-deploy" >}}) over
```

Reply:
I don't get this change

Reply:
this is very small nit. i am going to approve and merge